hoomd/md/pytest/test_methods.py

```diff
@@ -207,14 +202,14 @@ def test_rattle_attributes_attached(simulation_factory,
                                     two_particle_snapshot_factory,
                                     method_base_params):
 
-    if not method_base_params.has_rattle:
+    if method_base_params.rattle_method is None:
         pytest.skip("RATTLE integrator is not implemented for this method")
 
     all_ = hoomd.filter.All()
     gyroid = hoomd.md.manifold.Gyroid(N=1)
```

Comment:
I would like to see at least one test in this file be parametrized on both the integration method and on the manifold class. I think this test, `test_rattle_attributes_attached` would be the best candidate.

Reply:
Another option would be the new test that needs to be written in my first comment.